Automattic/sensei
# Add dependency group eslint rule

Fixes #3942

### Changes proposed in this Pull Request

* Add dependency group eslint rule.

### Testing instructions

* Run `npm run lint-js` and make sure everything is correct.
* Remove a dependency group comment and make sure a warning will appear.
